src/som/interpreter/actors/Actor.java

```diff
@@ -239,25 +239,15 @@ public ForkJoinWorkerThread newThread(final ForkJoinPool pool) {
   public static final class ActorProcessingThread extends ForkJoinWorkerThread {
     public EventualMessage currentMessage;
     protected Actor currentlyExecutingActor;
-    protected final ObjectBuffer<SFarReference> createdActors;
-    protected final ObjectBuffer<ObjectBuffer<EventualMessage>> processedMessages;
+    ObjectBuffer<SFarReference> createdActors;
+    ObjectBuffer<ObjectBuffer<EventualMessage>> processedMessages;
```

Comment:
Why do these fields need to be mutable (i.e., why removing the `final`)?
And, why is `protected` not sufficient anymore?